modules/cluster-logging-collector-legacy-fluentd.adoc

```diff
@@ -90,7 +90,7 @@ To configure {product-title} to forward logs using the legacy Fluentd method:
     host
   </server>
 ----
-<1> Specify the default value of the auto-generated certificate common name (CN).
+<1> Client hostname
```

Comment:
Use "host name" per [RH Supplementary Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#_h) throughout this module (there are a few other instances). Note that the IBM SG does indeed suggest the opposite.

Reply:
Also add periods to the end of this description and any others that are missing it in this list, to maintain consistency here (since there are fragment/sentence mixtures already in use).

Reply:
Thanks. Updated.